packages/sdk/src/Base.ts

```diff
@@ -322,13 +322,8 @@ class Base {
    *```
    */
   public async getSignerAddress () {
```

Comment:
We can't throw here because the signer might not be connected when getting only calldata. Instead, now it'll throw in places where it expects an address and it either wasn't passed in or signer was not connected.
For example, in `getNativeTokenBalance`:

![DeepinScreenshot_select-area_20220112145751](https://user-images.githubusercontent.com/168240/149236331-09a07d01-d1aa-4b13-8912-e8b6a6dcc4fb.png)